.env.development

```diff
@@ -5,3 +5,6 @@ PORT=3060
 WEBSITE_URL=http://localhost:3000
 SESSION_SECRET=i&j@/V6Wx.`g>Aq?qQQF(>u)/Erm;3A=
 DEBUG=email
+ORDER_LIMIT_ON_ACCOUNT_PER_HOUR=100
+ORDER_LIMIT_ON_ACCOUNT_PER_HOUR_PER_COLLECTIVE=20
+ORDER_LIMIT_ON_IP_PER_HOUR=20
```

Comment:
Let's use `config/default.json` for that, this file is gonna be deprecated soon.